phpcs.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards for Co-Authors Plus">
+
+	<rule ref="WordPress-VIP" />
+	<rule ref="WordPress-Docs" />
+	<rule ref="WordPress-Extra" />
+
+	<arg value="s"/>
+	<arg name="extensions" value="php"/>
+	<file>.</file>
+
+	<exclude-pattern>*/node_modules/*</exclude-pattern>
+	<exclude-pattern>*/vendor/*</exclude-pattern>
+</ruleset>
```

Comment:
By enabling some custom properties (see https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/wiki/Customizable-sniff-properties#properties-strongly-recommended-to-be-set) then extra checks, such as textdomains and deprecations will also be run.